tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -260,13 +269,29 @@ public void export() throws IOException {
       // new ops to the graph for saving and restoring the variables.
       SaverDef saverDef = graph.saverDef();
 
+      GraphOperation initOp = null;
+      if (!functions.containsKey(JAVA_INIT_OP_SIGNATURE_KEY)) {
+        initOp = graph.addInitOp(true);
+      }
+
       MetaGraphDef.Builder metaGraphDef =
           metaGraphDefBuilder
               .setSaverDef(saverDef)
               .setGraphDef(graph.toGraphDef())
               .setMetaInfoDef(MetaInfoDef.newBuilder().addAllTags(Arrays.asList(tags)));
       functions.forEach((k, f) -> metaGraphDef.putSignatureDef(k, f.signature().asSignatureDef()));
 
+      if (!functions.containsKey(JAVA_INIT_OP_SIGNATURE_KEY)) {
+
+        metaGraphDef.putSignatureDef(
+            JAVA_INIT_OP_SIGNATURE_KEY,
+            SignatureDef.newBuilder()
+                .putOutputs(
+                    JAVA_INIT_OP_SIGNATURE_KEY,
+                    TensorInfo.newBuilder().setName(initOp.name() + ":0").build())
```

Comment:
You can now use `initOp.output(0).name()` instead

Reply:
I can't, since `initOp` is a NoOp, so trying to get an output that doesn't exist will error.